nrf_modem/include/nrf_modem_os.h

```diff
@@ -72,26 +81,52 @@ void *nrf_modem_os_alloc(size_t bytes);
  */
 void nrf_modem_os_free(void *mem);
 
+/**
+ * @brief Allocate a buffer on dedicated trace heap.
+ *
+ * @param bytes Buffer size.
+ * @return void*
```

Comment:
? missing description

Reply:
Thanks, added for all `_alloc` functions